scripts/qa_crowbarsetup.sh

```diff
@@ -280,11 +280,23 @@ function add_mount()
     local nfssrc="$2"
     local targetdir="$3"
     local zypper_alias="$4"
+
     if [ -n "${localreposdir_target}" ]; then
-        [ -n "${bindsrc}" ] && add_bind_mount "${localreposdir_target}/${bindsrc}" "${targetdir}"
+        if [ -z "$bindsrc" ]; then
+            complain 50 "BUG: add_mount() called with empty bindsrc parameter" \
```

Comment:
That's not correct, I already did a review and overhaul of those in #215.  Plus it would only throw errors in the `$localreposdir_target` case (which apparently I am the only one using because it was broken before), and in this case it's better to have an error which explains accurately what's going on, rather than to have a knock-on failure later on which is much harder to understand.  That's what was happening for me before.